scripts/build/genesis/genesis.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
```

Comment:
I suggest to use `set -e`.

Reply:
It's not a problem, it's linux docker container and this tool is only for linux docker container